dist/main.js

```diff
@@ -0,0 +1,39 @@
+const render = function (roster) {
+    const source = $('#roster-template').html()
+    const template = Handlebars.compile(source)
+    const newHTML = template({roster})
+    $('#team-container').empty().append(newHTML)
+}
+
+//why didn't this work?
+// $('#roster-template').on("error", "img", function() {
```

Comment:
I mentioned this in class. You can check out this stack overflow answer for more info --> https://stackoverflow.com/questions/18683503/jquery-on-image-error-not-working-on-dynamic-images/18683695

I would just use the built-in HTML attribute `onerror` for this.